speech/gen_dataVals_from_wave_viewer.m

```diff
@@ -107,6 +107,21 @@
             numUserEvents = 0;
         end
```

Comment:
Using try/catch makes it a bit more opaque why this would fail. Checking for the field makes it more explicit. (Note: this would replace lines 104-108 but I'm not able to select lines 104-106 in this PR!)

```suggestion
        if isfield(trialparams,'event_params') && isfield(trialparams.eventparams,'user_event_times')
            numUserEvents = length(trialparams.event_params.user_event_times);
        else
            numUserEvents = 0;
        end
```

Reply:
Fixed in commit f5c11adda99b4e82e9176dc0455422ab387b6bd5